lusidtools/cocoon/cocoon.py

```diff
@@ -1426,6 +1429,15 @@ def load_from_data_frame(
         .value
     )
 
+    Validator(
```

Comment:
So "property_columns" can now be a list of strings or a list of dictionaries is that right?

Reply:
Yep, I can see this clearly now.

Reply:
Or a mix of strings and dictionaries if you wanted to do that